sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -598,3 +598,5 @@ CREATE TABLE `erx_narcotics` (
    PRIMARY KEY (`id`)
   ) ENGINE=InnoDB;
 #Endif
+
+UPDATE `supported_external_dataloads` SET `load_filename`='2018-ICD-10-Code-Descriptions.zip' WHERE `load_filename`='2018-ICD-10-Code-Dedcriptions.zip' AND `load_release_date`='2017-10-01';
```

Comment:
Actually, let me think about the best placement of this line some more.(am leaning towards this going right above the block above and changing the IfNotRow4D element back to 2018-ICD-10-Code-Descriptions.zip, but need to think about it some more)

Reply:
ok, i dbl checked the md5sum and it's good to go